apps/docs/src/app/core/component-docs/card/examples/card-example.component.html

```diff
@@ -22,25 +22,16 @@ <h2 fd-card-title>Card Title</h2>
 <br />
 
 <h3>Card header can be at the bottom as well</h3>
-<fd-card>
-    <fd-card-content>
-        <ul fd-list [noBorder]="true">
-            <li fd-list-item>
-                <span fd-list-title>
-                    item 1
-                </span>
-            </li>
-            <li fd-list-item>
-                <span fd-list-title>
-                    item 2
-                </span>
-            </li>
-        </ul>
-    </fd-card-content>
-    <fd-card-header>
-        <h2 fd-card-title>Card Title</h2>
-    </fd-card-header>
-</fd-card>
+<div class="fd-docs-card-fix-width">
+    <fd-card>
```

Comment:
why do you need a wrapper div? you can apply a class directly on the fd-card

Reply:
removed extra div